packages/components/src/local/molecules/json-editor/helpers/setupEditor.ts

```diff
@@ -21,36 +23,47 @@ const setupEditor = (editor: Editor | null, schema: any, ref: RefObject<HTMLDivE
   const promptBeforeDelete = 'prompt_before_delete'
   if (schema && schema.type && ref.current) {
-    if (schema.properties.location) {
-      setTimeout(() => {
-        const hiddenStores = document.querySelectorAll('div[data-schemapath$="hiddenStore"]') as NodeListOf<HTMLDivElement>
-        Array.from(hiddenStores).forEach(hiddenStores => {
-          hiddenStores.style.display = 'none'
-        })
+    // if (schema.properties.location) {
+    //   setTimeout(() => {
+    //     const hiddenStores = document.querySelectorAll('div[data-schemapath$="hiddenStore"]') as NodeListOf<HTMLDivElement>
+    //     Array.from(hiddenStores).forEach(hiddenStores => {
+    //       hiddenStores.style.display = 'none'
+    //     })
 
-        const locationSections = document.querySelectorAll('div[data-schemaid="locationArea"]') as NodeListOf<HTMLDivElement>
-        Array.from(locationSections).forEach(locationSection => {
-          // do not manipuate processed textarea
-          if (locationSection.childNodes.length === 3) {
-            return
-          }
+    //     const locationSections = document.querySelectorAll('div[data-schemaid="locationArea"]') as NodeListOf<HTMLDivElement>
+    //     Array.from(locationSections).forEach(locationSection => {
+    //       // do not manipuate processed textarea
+    //       if (locationSection.childNodes.length === 3) {
+    //         return
+    //       }
 
-          locationSection.style.display = 'flex'
-          locationSection.style.alignItems = 'center'
+    //       locationSection.style.display = 'flex'
+    //       locationSection.style.alignItems = 'center'
 
-          const textArea = locationSection.firstChild as HTMLDivElement
-          textArea.style.width = '30%'
+    //       const textArea = locationSection.firstChild as HTMLDivElement
+    //       textArea.style.width = '30%'
 
-          const editButton = document.createElement('button')
-          editButton.innerText = 'Edit'
-          editButton.name = 'editInLocation'
-          editButton.classList.add('btn', 'btn-secondary', 'json-editor-btn-add', 'json-editor-btntype-add')
-          editButton.style.height = '40px'
-          editButton.style.transform = 'translateX(-60px)'
-          editButton.addEventListener('click', () => editCallback())
-          locationSection.appendChild(editButton)
-        })
-      })
+    //       const editButton = document.createElement('button')
+    //       editButton.innerText = 'Edit'
+    //       editButton.name = 'editInLocation'
+    //       editButton.classList.add('btn', 'btn-secondary', 'json-editor-btn-add', 'json-editor-btntype-add')
+    //       editButton.style.height = '40px'
+    //       editButton.style.transform = 'translateX(-60px)'
+    //       editButton.addEventListener('click', () => editCallback())
+    //       locationSection.appendChild(editButton)
+    //     })
+    //   })
+
+    // test custom editor
```

Comment:
at some point you'll have to delete this please @lebaphi

Reply:
so when I removed this block, we should update the schema to use format `location` and type `string` to use the new editor